src/sql/azdata.proposed.d.ts

```diff
@@ -399,5 +399,14 @@ declare module 'azdata' {
 	export interface TaskInfo {
 		targetLocation?: string;
 	}
+
+	export enum ExtractTarget {
```

Comment:
does this need to be added in azdata or can this be in mssql.d.ts instead?

Reply:
I have a TODO in projectsController regarding the same. It can surely move to mssql.d.ts but that file just has const enums and I am not sure how will I be able to work around that for now. Hence deferring it for now. Hope that should be ok.

Reply:
Please move it to the mssql.d.ts. azdata.*.d.ts is ONLY for the extension API provided by core ADS. 

What do you mean by "how you will work around it"? Is there some issue with the const enums in that file?

Reply:
Done.